pymemcache/client/base.py

```diff
@@ -309,17 +309,19 @@ def set_many(self, values, expire=0, noreply=None):
                    self.default_noreply).
 
         Returns:
-          If no exception is raised, always returns True. Otherwise all, some
-          or none of the keys have been successfully set. If noreply is True
-          then a successful return does not guarantee that any keys were
-          successfully set (just that the keys were successfully sent).
+          Empty list
```

Comment:
doesn't this now return a list of failed sets.

Reply:
thx! i fixed it.